src/MonitorProfiler/MainProfiler/MainProfiler.h

```diff
@@ -32,3 +42,4 @@ class MainProfiler final :
 private:
     std::unique_ptr<CommandServer> _commandServer;
```

Comment:
Out of curiosity, why do we have multiple `private` keywords?

Reply:
Because @wiktork added fields from his PR to the bottom when I added them to the top with the initial merge. We should agree to some standard, maybe enforced by something like an *.editorconfig file.